### PR TITLE
Weekly Reflection 13

### LINKED ISSUE #20: Weekly REflection 13
This is for Weekly Reflection 13.

I'd like suggestions on formating my document. Is all the information displayed in good ways, am I using markdown correctly, and am I using it all to the best that I can?

### PR DESCRIPTION
**Pull request by HadesGeneral**
Spell checked it all and so I can make a pull request to link the new issue.

### CODE REVIEW

File: Reflections/Week_13_Reflection.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 1. What have you accomplished this week? (Please list each accomplishment, and explain) 
-  - I was able to conncect the Linear Actuator to the currently built system. (D4.5 shows this.)
-  - I was able to get it to work together with only a few hickups, since they were built sepertly, so new parts needed to be intergrated or moved around. (D4.5 shows this.)
+  - I was able to connect the Linear Actuator to the currently built system. (D4.5 shows this.)
```
Comment (belainehi) [inline Reflections/Week_13_Reflection.md:2]: It's great that you connected everything together. Does the remote control app work the same way?

File: Reflections/Week_13_Reflection.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+  - I was able to get it to work together with only a few hiccups, since they were built separately, so new parts needed to be integrated or moved around. (D4.5 shows this.)
   - I was able to test and show off the system working in it's entirety. (D4.5 shows this.)
 2. What challenges or difficulties did you face? If you solved them, how? If not, what have you learned so far? Have you sought help or other resources?
-  - I had trouble figuring out where to move the Linear Actuator's motor since I couldn't have it hang off. And so i moved it to under the Linear Actuator, and right beside the Lego Brain. I achived this by transfering the rotational movement of the motor with a axle and gear setup. (D4.5 shows this.)
+  - I had trouble figuring out where to move the Linear Actuator's motor since I couldn't have it hang off. And so, I moved it to under the Linear Actuator, and right beside the Lego Brain. I achieved this by transferring the rotational movement of the motor with an axle and gear setup. (D4.5 shows this.)
```
Comment (belainehi) [inline Reflections/Week_13_Reflection.md:6]: I watched the video and it's great that your system is able to change directions along with the weight. In you D4.5 can you specify how much weight you tested your system with? And how much the whole system weighs with the new changes?

File: Reflections/Week_13_Reflection.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
   - make a better platform for the weight on the Linear Actuator.
-    - Specific: The platform on the linear actutor currently can not support the weight correctly. It needs duck tape as extra support, and even then it isn't as strcutually sound. So making it where the weight can lay down without falling off when using the system will be the goal.
-    - Measurable: If the weight doesn't fall off or it doesn't move as easily loying down, then I have achived the goal.
+    - Specific: The platform on the linear actuator currently cannot support the weight correctly. It needs duct tape as extra support, and even then, it is not as structurally sound. So, making it where the weight can lay down without falling off when using the system will be the goal.
+    - Measurable: If the weight does not fall off or it does not move as easily lying down, then I have achieved the goal.
     - Achievable: This is kind achievable. The hard part is going to be making sure I don't change the setup to much so that it still works as intended.
```
Comment (belainehi) [inline Reflections/Week_13_Reflection.md:11]: You must give an estimate of how much time you plan to spend working on this, and when you expect results for all your S.M.A.R.T. goals.

File: Reflections/Week_13_Reflection.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
-    - Measurable: If the button is pressed and it stops functions for  amoment, then makes the skateboard back up then this shows I have done it. 
-    - Achievable: This is sort of, becuase the coding is semi-unformilar so coding this will be hard. I also don't want to break the sytem I have built so far, so I am wary for it all so far.
+    - Specific: As a stretch goal I would like to add a bump sensor. This is a button like part that can integrate into the Lego Mindstorms. This will allow me to make a program that when the button is pressed it will shut off all functions then make the motors back up the skateboard. As an emergency backup system of sorts. 
+    - Measurable: If the button is pressed and it stops functions for a moment, then makes the skateboard back up then this shows I have done it. 
+    - Achievable: This is sort of because the coding is semi-unfamiliar so coding this will be hard. I also do not want to break the system I have built so far, so I am wary for it all so far.
 4. What resources will you use to accomplish your goals for the upcoming week? (please list out the resources)
   - Current system
```
Comment (belainehi) [inline Reflections/Week_13_Reflection.md:21]: You should list out your components as well as anything you'll use for testing.

**Comment by jazzij** [comment]
Deliverable 4 document should only contain the documentation for your Low Fidelity prototype. Nothing submitted in Spring A will be graded again in Spring B. Anything more recent should be in its own implementation branch and document, organized in a reasonable manner, such as Week by Week process or Component by Component (Showing evolution of component development). Please rearrange your Semester B documentation accordingly

**Comment by belainehi** [comment]
For your testing, I recommend  Functional Testing (Both Unit and Integration) and End to End testing. Possible component groups to do a unit test over: 

Group 1: NXT Brain and Remote Control 
Group 1.1: NXT Brain, Skateboard and Remote Control
Group 2: Linear Actuator and Remote Control 
Group 3: Linear Actuator with weight and Remote Control
**(Add more components groups that make sense)**

It is expected that you will come up with component groups that make sense for your project and that you will document your testing for D5. You have already been testing as you've been building your project so it shouldn't be too hard.

**Comment by HadesGeneral** [comment]
Can I merge this?

**Comment by belainehi** [comment]
@HadesGeneral you can merge this. The Week_13_Reflection.md file will cause a merge conflict though, you might need to use command line. Can you come to lab on Saturday?

**Comment by HadesGeneral** [comment]
@belainehi I don't know how to use the command line. I have been using strickly the browser. When is the lab, there's a good chance I can not make it?

**Pull request merged**